### DISCUSSION TITLE
Best way to use models mapped to tutorials

### DISCUSSION THREAD

**Discussion by dbericat** [Project-MONAI/tutorials#334] [Q&A] [answered]
Hi @wyli @Nic-Ma @ericspod , 

As we think about how to put together MONAI Deploy examples and tutorials, we'd like to reuse some of the models mapped to these tutorials. 

In particular, we couldn't find the model for https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb

New area for me, so how does this work?

Thanks in advance!

**Comment by Nic-Ma** [comment]
Hi @dbericat ,

Excited to see the MONAI deploy progress.
May I know what do you mean `could't find the model`? You want the network or trained model weights?
The network is `UNet` in MONAI source code.
We don't maintain some trained weights so far.

Thanks.

**Comment by dbericat** [comment, marked as answer]
I think we were trying to find the trained model weights. Otherwise we can't package it in a MONAI Deploy App. @MMelQin @gigony @rahul any comments?

**Reply by rijobro** [reply]
the tutorial uses `torch.save`, so as long as you run the tutorial, you'll save the network weights under `os.path.join(root_dir, "best_metric_model.pth")`, where:

```python
directory = os.environ.get("MONAI_DATA_DIRECTORY")
root_dir = tempfile.mkdtemp() if directory is None else directory
```
so make sure `MONAI_DATA_DIRECTORY` is set as an environmental variable.

**Reply by dbericat** [reply]
Thanks Richard!
